app/posts/apply-for-teacher-training/2020-06-18-international-candidates.md

```diff
@@ -0,0 +1,244 @@
+
+Our [initial design for residency status](/apply-for-teacher-training/residency-status/) assumed that candidates would know which visa they require.
+
+However, research with international candidates suggested that it’s too early in the process to ask this question. Candidates have not necessarily considered these details yet, and financial circumstances (and therefore visa requirements) might change between application and acceptance.
+
+It may be useful for international applicants to know which providers can sponsor a visa, but this information should be available earlier in the journey – possibly on Find.
+
+### User need
+
+{% from "user-needs/macro.njk" import appUserNeeds %}
+{{ appUserNeeds({
+  items: collections['user-need'] | slugs(['AN025'])
+}) }}
+
+### Hypothesis
+
+If we allow international candidates to defer entering details about visas
+Then they can continue with their application
+We’ll know this works when the proportion of international candidates who submit a complete application compared to those who start, is comparable to the same value for domestic candidates.
+
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+{{ appScreenshots({
+  headingLevel: 3,
+  items: [{
+    text: "Personal details",
+    caption: "Ask the nationality question in a separate step and remove the question about languages spoken."
+  }, {
+    text: "What is your nationality?",
+    caption: "* British and Irish – no further questions are asked\n* Other – candidate asked to select nationality from structured list\n* I have multiple nationalities – candidate can give details as free text"
+  }, {
+    text: "Do you have the right to work or study in the UK? (Yes)",
+    caption: "Candidates who are not British or Irish are asked to give details about their residency status, of they know it. They can give details of their residency, or indicate that this is something they will need to consider later."
+  }, {
+    text: "Do you have the right to work or study in the UK? (Not yet)",
+    caption: "They can also indicate that this is something they will need to consider later by selecting ‘Not yet’ or ‘I do not know’."
+  }, {
+    text: "Reviewing personal details"
+  }]
+}) }}
+
+## International addresses
+
+This section remains unchanged from the [original design for capturing international addresses](/apply-for-teacher-training/international-address/). We can derive a candidate’s domicile from the information given in this section.
+
+## English language proficiency
+
+Our [original design for English language proficiency](/apply-for-teacher-training/apply-launch/#30-personal-details) asked candidates to tell us if their main language was English. If yes, we asked them if they spoke any other languages. If no, we asked for details of any English language qualifications they have, as well as other languages spoken.
```

Comment:
```suggestion
Our [original design for English language proficiency](/apply-for-teacher-training/apply-launch/#30-personal-details) asked candidates to tell us if their main language is English. If yes, we asked them if they spoke any other languages. If no, we asked for details of any English language qualifications they have, as well as other languages spoken.
```

My main language was English 
vs 
My main language is English

Reply:
Oops, fixed!